ui/widgets/nuxeo-html-editor.js

```diff
@@ -55,121 +51,90 @@ import './alloy/alloy-ocean.js';
 
     static get properties() {
       return {
-        element: Object,
-
+        /**
+         * HTML value
+         */
         value: {
           type: String,
           notify: true,
-          observer: '_valueChanged',
         },
 
+        /**
+         * Placeholder content to be displayed when empty
+         */
         placeholder: {
           type: String,
           value: 'Type here...',
         },
 
-        readOnly: Boolean,
+        /**
+         * When set the content will be read only.
+         */
+        readOnly: {
+          type: Boolean,
+          value: false,
+          reflectToAttribute: true,
+        },
 
-        hideToolbars: Boolean,
+        _editor: {
+          type: Object,
```

Comment:
nit: would add readOnly: true

Reply:
Not a fan of using `readOnly` with private properties as it's mostly meant to prevent downward data flow aka preventing the host from modifying the property. Since HTML attributes can't start with `_` this is already the case.

Reply:
I can live with that. It was just a nitpick. 👍

Reply:
We need to clarify and align on this since we've been doing it as part of elements migration when we believe it makes sense.